Tests/GRPCCodeGenTests/Internal/Translator/ServerCodeTranslatorSnippetBasedTests.swift

```diff
@@ -0,0 +1,398 @@
+/*
+ * Copyright 2023, gRPC Authors All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import XCTest
+
+@testable import GRPCCodeGen
+
+extension SnippetBasedTranslatorTests {
```

Comment:
I'm pretty sure this is not the best way to add the SnippetBased tests for the ServerCode translator. There are some functions that are used by both type alias and server code tests. Should they be the only primary components of the "SnippetBasedTranslatorTests", and the tests part of extensions in different files?

Reply:
Just the `makeCodeGenerationRequest` function actually